src/views/Login.vue

```diff
@@ -1,5 +1,15 @@
 <template>
     <v-container>
+        <v-overlay :value="true" v-if="loading">
```

Comment:
Why has this code been copied and pasted between views instead of using a single component?

Reply:
The loading indicator isn't even centered...